### DISCUSSION TITLE
cohesive elements for hexahedral mesh

### DISCUSSION THREAD

**Discussion by ravikiranbollineni** [neperfepx/neper#481] [Q&A] [answered]
Hello Romain,

I am trying to create cohesive elements at the grain boundaries for a 3D model. when I am using tetrahedron elements neper is generating without any error but when i use for hexahedral elements neper is giving me the following error:

Info   :     > Making mesh discontinuous...
Segmentation fault (core dumped)

could you help in this regards?

Thank you,
Ravi.

**Comment by rquey** [comment, marked as answer]
It's not available for hex meshes, but Neper2CAE seems to do the job... See https://neper.info/others.html#id1.
